scribejava-ahc/src/main/java/com/github/scribejava/core/async/ahc/AhcHttpClient.java

```diff
@@ -0,0 +1,63 @@
+package com.github.scribejava.core.async.ahc;
```

Comment:
It's not in 'core' module now. And I think, "async" is not the valuable keyword already.
So as for me the package name should be just
com.github.scribejava.httpclient.ahc
to reflect maven arifact id